### PR TITLE
add ebpf user group

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR adds eBPF user group to the community drop down on the nav bar

### CODE REVIEW

File: src/data/shared/header-menu.js [hunk 1]
```diff
@@ -128,6 +128,13 @@ const headerMenu = [
         },
         to: 'https://ebpf.io/slack',
       },
+      {
+        title: {
+          en: 'eBPF User Group',
+          'fr-fr': 'eBPF User Group',
```
Comment (xmulligan) [inline src/data/shared/header-menu.js:134]:
```suggestion
          en: 'eBPF Meetup',
          'fr-fr': 'eBPF Meetup',
```

Comment (hacktivist123) [reply src/data/shared/header-menu.js:134]: I'm afraid I have to disagree with this. 

Not only meetups can happen in the User Group, but a meetup is just one part of the User Group, other events happen in the User Group and this is how I've seen other OSS communities name it. 

It shouldn't be eBPF Meetup

**Comment by xmulligan** [comment]
Can you also add it in the footer as a logo? 
Screenshot 2023-09-18 at 11 32 26

**Comment by hacktivist123** [comment]
> Can you also add it in the footer as a logo? Screenshot 2023-09-18 at 11 32 26

@xmulligan I think the issue would be getting the icon for meetup, I think those are custom icons designed for the ebpf.io page. I can give it a go and see if it matches with eh existing Icon

File: src/data/shared/header-menu.js (hunk 1 above)
Comment (xmulligan) [reply src/data/shared/header-menu.js:134]: I spoke with Daniel about this too and he wants to go with Meetup because user group sounds old fashioned

Comment (hacktivist123) [reply src/data/shared/header-menu.js:134]: alright then! sounds good

**Comment by hacktivist123** [comment]
> > Can you also add it in the footer as a logo? Screenshot 2023-09-18 at 11 32 26
> 
> @xmulligan I think the issue would be getting the icon for meetup, I think those are custom icons designed for the ebpf.io page. I can give it a go and see if it matches with the existing Icons

@xmulligan just investigated this and it seems like we'll need to have a custom meetup icon created for this purpose by pixel point because it didn't match the styling when I tried it. Should I escalate this to Pixel point?

**Comment by xmulligan** [comment]
button was added here https://github.com/ebpf-io/ebpf.io-website/pull/489

Just realized can you also add it to the events drop down?

**Comment by hacktivist123** [comment]
> button was added here #489
> 
> Just realized can you also add it to the events drop down?

done @xmulligan

**Pull request merged**